### PR TITLE
fix(installer): Fix Nginx not starting when statistics service is disabled

### PR DESCRIPTION
**Pull request by warber**
PR to fix problems with having statistics service disabled via the helm chart

### CODE REVIEW

File: installer/manifests/keptn/values.yaml [hunk 1]
```diff
@@ -365,7 +365,7 @@ lighthouseService:
       cpu: "200m"
 
 statisticsService:
-  enabled: false
+  enabled: true
```
Comment (christian-kreuzberger-dtx) [inline installer/manifests/keptn/values.yaml:368]: Does it make sense to have it enabled by default? Or was this an accidental change :D

Comment (warber) [reply installer/manifests/keptn/values.yaml:368]: Actually, having it set to fals was an accidental change, which caused us to discover the problem with nginx :D :dagger:

Comment (warber) [reply installer/manifests/keptn/values.yaml:368]: but I agree, we should think about whether false should be the default, actually....
@thisthat ?

**Review by RealAnna — APPROVED** [review]

**Comment by warber** [comment]
This will fix the master branch again.
@christian-kreuzberger-dtx we'll discuss the option of disabling statistics service per default and do it in a follow up. I think everyone kind of agrees to that proposal...

**Pull request merged**